digdag-tests/src/test/resources/acceptance/s3/s3_wait.dig

```diff
@@ -0,0 +1,6 @@
+
++wait:
+  s3_wait>: ${path}
```

Comment:
Is it possible to add some more tests for other commands (`region`, `bucket`, ...) and SSE-C commands (`aws.sse-key`, ...). We'd better test them before releasing them.